pages/modules/[module].tsx

```diff
@@ -198,59 +198,101 @@ const ModulePage: NextPage<ModulePageProps> = ({
                     </button>
                   )}
                 </div>
-                <h2 className="text-2xl font-bold mt-4">
-                  Dependencies{' '}
-                  <small className="text-sm font-normal text-gray-500">
-                    (of version {selectedVersion})
-                  </small>
-                </h2>
-                <div>
-                  <ul className="mt-4">
-                    {versionInfo.moduleInfo.dependencies.map((dependency) => (
-                      <Link
-                        key={dependency.module}
-                        href={`/modules/${dependency.module}/${dependency.version}`}
-                      >
-                        <li className="border rounded p-2 mt-2 flex items-center gap-4 hover:border-gray-800">
-                          <div className="rounded-full border h-14 w-14 grid place-items-center">
-                            {dependency.version}
-                          </div>
-                          <div>{dependency.module}</div>
-                        </li>
-                      </Link>
-                    ))}
-                    {versionInfo.moduleInfo.dependencies.length === 0 && (
-                      <span>No dependencies</span>
-                    )}
-                  </ul>
+                <div className="mt-4">
+                  <h2 className="text-2xl font-bold mt-4">Dependency graph</h2>
                 </div>
-                <h2 className="text-2xl font-bold mt-4">Dependents</h2>
-                <div>
-                  <ul className="mt-4">
-                    {shownReverseDependencies.map((revDependency) => (
-                      <Link
-                        key={revDependency}
-                        href={`/modules/${revDependency}`}
+                <div className="mt-4">
+                  <details open>
+                    <summary>
+                      <span role="heading" aria-level={2} className="text-1xl mt-4">
+                        <span className="font-bold">Transitive</span> ({versionInfo.moduleInfo.dependencies.filter((d) => !d.dev).length || 'None'}){' '}
```

Comment:
aren't these direct dependencies, instead of transitive dependencies? "transitive dependencies" means your deps + deps of your deps + deps of deps of your deps + ...

Reply:
I called them "transitive" because they are transitive from the perspective of a package consumer, who is evaluating whether to add it to their project; you are right that these are "direct" dependencies from the perspective of the package itself.

I can change it back if this is confusing? Do you think "Direct" dependencies of the package is a clear expression, or is there a risk users could think those are "peer" dependencies which also need to be added to their project? Honest question and I am happy to go either way.

Reply:
Upon further reflection, it's not the full closure of "transitive" dependencies, so either way you're right, we shouldn't call it this.

Reply:
@Wyverald I've changed it to _Direct_ for now, unless we can come up with something better 👍🏻

Reply:
Direct is the term I would expect here